MoreLinq/HasDuplicates.cs

```diff
@@ -0,0 +1,44 @@
+namespace MoreLinq
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    static partial class MoreEnumerable
+    {
+        /// <summary>
+        ///   Checks if an enumerable of T contains duplicates
+        /// </summary>
+        /// <param name="source">The enumerable of T to validate.</param>
+        /// <typeparam name="T">The item type</typeparam>
+        /// <returns>true if any element of the source enumerable is duplicated, false otherwise</returns>
```

Comment:
there is no need to explicit that is an "enumerable of T", just "sequence" is enough. Take a look in other summaries to check this and other patterns used in documentation.